input/input.go

```diff
@@ -0,0 +1,252 @@
+/*
+Package Input reads config file and populates the data in Config struct.
+*/
+package input
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/maplelabs/github-audit/logger"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	DefaultBranch = "master"
```

Comment:
currently in GitHub, default branch is changed to "main". Check once.

Reply:
added both master and main branches as default